guides/common/modules/proc_performing-post-upgrade-tasks.adoc

```diff
@@ -1,6 +1,9 @@
 [id="performing-post-upgrade-tasks_{context}"]
 = Performing post-upgrade tasks
 
+* If you created backups of any custom configuration as described in xref:backing-up-custom-{project-context}-server-configuration-before-upgrading[], restore your custom changes.
+Note that if you make manual edits to the DNS or DHCP configuration files, you must restart {Project} services with the `{foreman-maintain} service restart` command.
```

Comment:
Given management of DNS/DHCP is disabled, you won't have to restore the configs or restart the services.

So it's only the templates you might want to restore.